MobSF/settings.py

```diff
@@ -249,21 +251,8 @@
 
     # ==========DECOMPILER SETTINGS=================
 
-    DECOMPILER = 'cfr'
-
-    # Three Decompilers are available
-    # 1. jd-core
-    # 2. cfr
-    # 3. procyon
-
-    # ==============================================
-
-    # ==========Dex to Jar Converter================
-    JAR_CONVERTER = 'd2j'
+    DECOMPILER = 'jadx'
```

Comment:
we don't need a settings if it cannot be configured

Reply:
agree